src/main/scala/us/diempham/taskman/web/services/User.scala

```diff
@@ -1,6 +1,20 @@
 package us.diempham.taskman.web.services
 
+import scala.collection.mutable
+
 object User {
   case class User(id: String, email: String, password: String)
 
+  val database: mutable.Map[String, User] = mutable.Map.empty[String, User]
```

Comment:
This should be private.  IT is exposing mutable state to the rest of the application.